app/handlers/blocks/block.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime
+
+from structlog import get_logger
+
+from app.data_model.progress_store import CompletionStatus
+from app.questionnaire.location import InvalidLocationException
+from app.questionnaire.path_finder import PathFinder
+from app.questionnaire.placeholder_renderer import PlaceholderRenderer
+from app.questionnaire.questionnaire_store_updater import QuestionnaireStoreUpdater
+from app.questionnaire.router import Router
+from app.questionnaire.schema_utils import transform_variants
+
+logger = get_logger()
```

Comment:
It doesn't look like logger is used

Reply:
It's used in the `set_started_at_metadata` method.